test/performance/README.md

```diff
@@ -1,128 +1,117 @@
+   - OPENSHIFT_CLUSTER: set to `true` for Kubernetes clusters (*default value: "false"*)
+   - tuningSet: workload generation function (*default value: "RandomizedLoad"*)
 
+   To learn more about clusterloader2 tuningSet, see the following: [tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set) and [configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
 
-## Collect the measurements 
 
-To collect the creation and status update for a k8s object proceed as following:  
 
-1. Create a kubeconfig with contexts for the `wds1`, `its1` and `wec1`:
 
-For example: 
+4. Deploy your first workload:
 
-```bash
-$ kubectl config get-contexts
-CURRENT   NAME     CLUSTER                                          AUTHINFO                                                                 NAMESPACE
-          its1     its1-cluster                                     its1-admin                                                                default
-          wec1     <url>:port#                                      <defaul-value>                                                            default
-*         wds1     wds1-cluster                                     wds1-admin                                                                default
-```
+   First, use a kubeconfig that includes contexts for WDS spaces and set its current context to a target WDS space (e.g., `wds1`):
 
-Use the following command to rename the default context name for `wec1` workload execution cluster:
+   ```bash
+   $ kubectl config use-context wds1
+   Switched to context "wds1".
 
-```bash 
-$ kubectl config rename-context <default-wec1-context-name> wec1
-```
+   $ kubectl config get-contexts
+   CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+            cluster1        kind-cluster1   kind-cluster1   
+            cluster2        kind-cluster2   kind-cluster2   
+            its1            its1-cluster    its1-admin      
+            kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *        wds1            wds1-cluster    wds1-admin      default
+            wds2            wds2-cluster    wds2-admin      default
+   ```
 
-2. Configure the following parameters in the `main` function in the `metrics_collectory.py` script
 
-```bash
-    kubeconfig = "kscore-config"  # path to the kubeconfig file
-    ns = "stress-test"  # name of the namespace where the workload are generated 
-    mypath="data/"  # path to the directory for the output files
-    run_ID="1" # run ID for your experiment 
-    freq="1"   # workload frequency parameter set in your experiment 
-    numPods=20 # total number of workload objects set in your experiment
-    method = "exp-RandomizedLoad" # clusterloader method used to generate your traffic
-    watch_interval = 3600  # watch for events that to occur within time interval threshold
```

Comment:
Why is this content being simply deleted? Are the defaults always what the user wants? Are these settings irrelevant?

Reply:
That content is obsolete. I need to re-write the scripts and consequently update the instructions with a NEW PR. I am planning to push this PR by this Thursday the latest.